mlrun/runtimes/base.py

```diff
@@ -196,6 +196,9 @@ def set_label(self, key, value):
         self.metadata.labels[key] = str(value)
         return self
 
+    def set_categories(self, categories: List[str]):
+        self.metadata.categories = categories
```

Comment:
`mlrun.utils.helpers.as_list` to ensure the input is indeed a list

Reply:
Done